src/libraries/Common/tests/System/Security/Cryptography/MLKemTests.cs

```diff
@@ -464,7 +464,7 @@ public static void ImportEncryptedPkcs8PrivateKey_WrongAlgorithm()
                 gJoH9z0+/Z9WzLU8ix8F7B+HWwRhib5Cd6si+AX6DsNelMq2zP1NO7Un416dkg==");
 
             Assert.Throws<CryptographicException>(() =>
-                MLKem.ImportEncryptedPkcs8PrivateKey("PLACEHOLDER", new ReadOnlySpan<byte>(ecP256Key)));
+                MLKem.ImportEncryptedPkcs8PrivateKey("PLACEHOLDER", ecP256Key));
```

Comment:
I didn't have to change that many tests because most of the tests used a `string` for the `password` parameter forces binding to a specific overload. I just changed this one since the span-wrapping served no purpose anymore.